amgut/templates/emperor.html

```diff
@@ -0,0 +1,288 @@
+      var md_headers = {% raw md_headers%};
+      var metadata = {% raw  metadata %};
+
+      var dm;
+
+      function init() {
+        // Initialize the DecompositionModel
+        dm = new DecompositionModel(name, ids, coords, pct_var,
+                                    md_headers, metadata);
+        // Initialize the EmperorController
+        ec = new EmperorController(dm, 'emperor-plot');
+      }
+
+      $(window).resize(function() {
+        ec.resize(window.innerWidth * 0.5, window.innerHeight * 0.7);
+      });
+
+      function animate() {
+        requestAnimationFrame(animate);
+        ec.render();
+      }
+
+      function rescale() {
+        var sv = ec.sceneViews[0];
+        //Shrink original points back to regular size and opacity
+        for (var i = 0 ; i < ec.sceneViews[0].decViews.scatter.markers.length; i++) {
+          var point = sv.decViews.scatter.markers[i];
+          point.material.transparent = true;
+          point.material.opacity = 0.4;
+          point.scale.x = 1;
+          point.scale.y = 1;
+          point.scale.z = 1;
+        }
+
+        //TRIPLE SIZE OF BARCODE POINT
+        var sample = sv.scene.getObjectByName($("#barcode-fold").val());
+        sample.scale.x = 3;
+        sample.scale.y = 3;
+        sample.scale.z = 3;
+        point.material.opacity = 1;
+        sample.material.transparent = false;
+      }
+
+
+      $(document).ready(function(){
+        init();
+        animate();
+        var sv = ec.sceneViews[0];
+
+        $("#barcode-fold").on("change", function () {
+          fold_change();
+          rescale();
+        });
+
+        $("#meta-cat-fold").on("change", function () {
+          fold_change();
+          cat = $("#meta-cat-fold").val().split("-")[0];
+          switch(cat) {
+            case 'age':
+              cat = 'age_cat';
+              break;
+            case 'bmi':
+              cat = 'bmi_cat';
+              break;
+            case 'sex':
+              cat = 'sex';
+              break;
+            default:
+            cat = 'all';
+          }
+          ec.colorController.$select.val(cat);
+          ec.colorController.$select.trigger("change");
+        });
+
+
+        //HIDE EDITING USING EMPEROR
+        $(function(){
+          //var barcodes = getParameterByName('barcodes').split(',');
+          ec.colorController.$header.hide();
+          ec._$tabsList.hide();
+          ec.colorController.$select.val('all');
+          ec.colorController.$select.trigger("change");
+
+          //CHANGE BG TO WHITE AND AXES TO BLACK
+          sv.drawAxesLabelsWithColor('#000000');
+          sv.drawAxesWithColor('#000000');
+          sv.renderer.setClearColor('#FFFFFF');
+
+          rescale();
+
+        //RECOLOR TO HIGHLIGHT POINTS
+          s = sv.decViews.scatter;
+          s.setGroupColor(0xFF0000, s.decomp.getPlottableByIDs([$("#barcode-fold").val()]));
+        });
+      });
+  }); // END REQUIRE.JS block
```

Comment:
Took the emperor template and added to it to create this page. That way we get emperor and all the other functionality we want on a single page.